src/_includes/layouts/default.njk

```diff
@@ -10,7 +10,6 @@
   <body>
     {% include "partials/header.njk" %}
     <main>
-      <h1>{{ title }}</h1>
```

Comment:
Well spotted about the duplicate `h1` tags!

I think the solution is not to remove this line from the shared layout, though. This is where the "title" defined in each page goes, and if we remove it many pages will not have an `h1` tag at all. For example, https://www.zinc.coop/product-and-services/, defined in https://github.com/zinc-collective/www.zinc.coop/blob/development/src/product-and-services/index.md?plain=1, would no longer show the "Products and Services" title.

I think a better solution is to fix the buggy home page implementation: it is missing a "title" in its metadata, which causes this to be an empty `h1` tag, and then adds an extra top-level heading to compensate.

In short:
* restore this line in the shared template
* in https://github.com/zinc-collective/www.zinc.coop/blob/development/src/index.md?plain=1
  * add `title: The Zinc Collective` in the metadata section for the page
  * remove extra header from line 4
  
Optionally, it might be a good idea to double-check that all of our pages have a `title` defined.

Reply:
Neat!